content/blog/2020/03/2020-03-17-ui-plugins.adoc

```diff
@@ -0,0 +1,709 @@
+---
+layout: post
+title: "Hands On: Beautify the user interface of Jenkins reporter plugins"
+tags:
+- ui
+- ux
+- plugins
+- developer
+author: uhafner
```

Comment:
It is probably the best one from the clickbait PoV. https://github.com/jenkins-infra/jenkins.io/blob/02018f49ac34e0511bbfe02cdf7bd26bee0c6871/content/images/post-images/2020/01-ui-plugins/card.png might be better content-wise

```suggestion
author: uhafner
opengraph:
  image: /images/post-images/2020/01-ui-plugins/jobs.png
```

![image](https://user-images.githubusercontent.com/3000480/76851137-068c9800-6849-11ea-850b-0ff72d96e2e3.png)

Reply:
Well, the jobs image actually has no new UI in it.

Reply:
![Bildschirmfoto 2020-03-17 um 15 11 43](https://user-images.githubusercontent.com/503338/76864641-b5d56900-6861-11ea-81ed-c1003e1334d4.png)

Reply:
What about that?

Reply:
Fine with me, thanks!

Reply:
@uhafner will you be able to add that opengraph reference?  This is a great blog post that I'd like to see on the jenkins.io site.  The opengraph references increase the rate at which people read the blog posts when they are promoted on twitter and other social media.

Reply:
Done, see https://github.com/jenkins-infra/jenkins.io/pull/2797/commits/ff8a11fa2292940565b3ee367d59c18abf0964e0.